web-app/web-frontend/script.js

```diff
@@ -72,12 +38,43 @@ function onSnapEnd () {
   board.position(game.fen())
 }
 
+function updateStatus () {
+  var status = ''
+
+  var moveColor = (game.turn() == 'b') ? "Black" : "White";
+
+  // checkmate?
+  if (game.in_checkmate()) {
+    status = 'Game over, ' + moveColor + ' is in checkmate.'
+  }
+
+  // draw?
+  else if (game.in_draw()) {
+    status = 'Game over, drawn position'
+  }
+
+  // game still on
+  else {
+    status = moveColor + ' to move'
+
+    // check?
+    if (game.in_check()) {
+      status += ', ' + moveColor + ' is in check'
+    }
+  }
+
+  $status.html(status)
+  $pgn.html(game.pgn())
```

Comment:
I am not super familiar with JS but I think this is where you can display the last move in GAN format.